### PR TITLE
Feature/excel import session attributes

### PR DESCRIPTION
**Pull request by kthomas-amzn**
Added support for importing session attributes via Excel.

attributename(n) - session attribute name
attributevalue(n) - session attribute value
enabletranslation(n) - determines whether the attribute value should be translated

Updated README and the Excel sample spreadsheet

### CODE REVIEW

**Review by rstrahan — COMMENTED** [review]
The diffs in import.vue are huge - I can really make head nor tail of the changes, so impossible for me to review. How come the changes are so extensive?  Are all these changes due only to this one feature?  Maybe Chris can make sense of it, otherwise I think we either need to (1) limit the diffs to only the changes related to this feature (I suspect maybe other changes crept in? Or (2) forget about code review and we just rely on testing.   
Speaking of testing, can you ass notes to say what testing you did to verify a) that yuou did not introduce any regression problems - ie imports of a) JSON and b) Excel files that worked before still work, and b) the the new session attribute support works as expected.

**Comment by rstrahan** [comment]
Sure thing Karl.. Thanks!

File: .gitignore [hunk 1]
```diff
@@ -80,3 +80,5 @@ temp/
 **/deployment/global-s3-assets
 **/deployment/regional-s3-assets
 **/deployment/open-source
+docs/excel_import/~$sample.xlsx
```
Comment (tabdunabi) [inline .gitignore:83]: - Aren't we including the `sample.xlsx` file with the solution (adding it to the GitHub repo) for the customers to know how to use the excel import feature?, why adding it to the `.gitignore` file?
- If you want to exclude the file, there is a typo: it should be  `docs/excel_import/sample.xlsx`, not `docs/excel_import/~$sample.xlsx`.

File: docs/excel_import/README.md [hunk 2]
```diff
@@ -27,6 +27,9 @@ Excel Column | Content Designer Field| Description
  cardimageurl | Card Image Url | The web address of an image that will appear on the response card
  displaytext1...displaytext(n) | Display Text | The text of the button. Each button text should be in a separate column numbered button1, button2, ...
  buttonvalue...buttonvalue(n) | Button Value| The value of the button. Each button value should be in a separate column numbered button1, button2, ...
+ attributename1...attributename(n) | Session Attribute Name | The name of the session attribute. Each attribute name should be in a separate column numbered attributename1, attributename2, ...
```
Comment (tabdunabi) [inline docs/excel_import/README.md:30]: Could you please provide steps to tests this feature (test case with required assets (e.g., sample inputs)), so our QA can added to our test cases for the coming release.

File: .gitignore (hunk 1 above)
Comment (kthomas-amzn) [reply .gitignore:83]: docs/excel_import/~$sample.xlsx is a temporary file that Excel creates.

File: docs/excel_import/README.md (hunk 2 above)
Comment (kthomas-amzn) [reply docs/excel_import/README.md:30]: The Excel file  in docs/excel_import/ has been updated with sample session attributes.

Comment (tabdunabi) [reply docs/excel_import/README.md:30]: Thanks

**Review by tabdunabi — APPROVED** [review]
LGTM.

**Pull request merged**